openage/convert/value_object/read/media/datfile/lookup_dicts.py

```diff
@@ -338,6 +340,10 @@
     102: "TECH_TOGGLE",       # d == research_id
     103: "TECH_TIME_MODIFY",  # a == research_id, if c == 0: d==absval else d==relval
 
+    -54: "UNKNOWN",  # 199: "UNKNOWN",
+    -55: "UNKNOWN",  # 200: "UNKNOWN",
+    -56: "UNKNOWN",  # 201: "UNKNOWN",
```

Comment:
-54 sure?

Reply:
Oh I forgot about this during experimenting with the values. I think the type is actually unsigned. That's also what genieutils uses for the type:

https://github.com/Tapsa/genieutils/blob/4b454e5b9f26a240078bb6a5c2bd3f514b3dc94b/src/dat/TechageEffect.cpp#L45